hrpc/scan.go

```diff
@@ -209,6 +230,27 @@ func (s *Scan) RenewalScan() bool {
 	return s.renewalScan
 }
 
+func (s *Scan) ScanStatsHandler() ScanStatsHandler {
+	return s.scanStatsHandler
+}
+
+func (s *Scan) ScanStats() *ScanStats {
+	return s.scanStats
+}
+
+func (s *Scan) ColumnFamilies() [][]byte {
+	cfs := make([][]byte, len(s.families))
+	for f, _ := range s.families {
+		cfs = append(cfs, []byte(f))
+	}
+	return cfs
+}
```

Comment:
I think we can leave out ColumnFamilies for now. My concern is that this is a little more expensive to add to the scan stats since it requires allocations and it could end up being a large slice.